sabnzbd.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "sabnzbd",
+    "release": "11.3-RELEASE",
+    "artifact": "https://github.com/RogerAirgood/iocage-plugin-sabnzbd.git",
+    "properties": {
+        "nat": 1,
+        "nat_forwards": "tcp(8080:8080)"
+    },
+    "pkgs": [
+	      "sabnzbdplus"
```

Comment:
Can we please correct the indentation for this line ?

Reply:
I've corrected the indent, thanks.